Applications/Games/Subnautica Below Zero/resources/fix.reg

```diff
@@ -0,0 +1,4 @@
+REGEDIT4
+        
+[HKEY_CURRENT_USER\Software\Wine\X11 Driver]
+"UseTakeFocus"="N"
```

Comment:
Not sure since the base Subanutica game was the only game I've tested that needed this registry tweak. But now I've removed that entry from both the main game and it's Below Zero expansion. And it no longer crashes on clicking the exit button in main menu. Guess either the newer version of wine or dxvk solves that. Will remove the regedit from both scrips.

Reply:
Ok.